closure/runtime.c

```diff
@@ -142,7 +139,7 @@ GC support stub routines
 
 
 
-static void *_Block_alloc_default(const unsigned long size, const bool initialCountIsOne, const bool isObject) {
+static void *_Block_alloc_default(uintptr_t size, const bool initialCountIsOne, const bool isObject) {
```

Comment:
Shouldn't this still be size_t though? I can see using the ptr types for the CAS.